apps/web/pages/[user].tsx

```diff
@@ -157,18 +214,21 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
       plan: true,
       away: true,
       verified: true,
+      allowDynamicBooking: true,
     },
   });
 
-  if (!user) {
+  if (!users.length) {
     return {
       notFound: true,
     };
   }
 
   const credentials = await prisma.credential.findMany({
     where: {
-      userId: user.id,
+      userId: {
+        in: users.map((user) => user.id),
```

Comment:
move to a constant 
`const usersIds = users.map((user) => user.id);`

`in: userIds,`